src/test/java/org/jabref/model/FieldChangeTest.java

```diff
@@ -0,0 +1,63 @@
+package org.jabref.model;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class FieldChangeTest {
+
+  private FieldChange Fieldchange;
+
+  @Test
+  void testHashCode() {
+    FieldChange fcNull = new FieldChange(null, null, null, null);
+    assertEquals(923521, fcNull.hashCode());
+  }
+
+  @Test
+  void testEquals() {
+    BibEntry entry = new BibEntry();
```

Comment:
We talked about this before in your other PR. No testing on equals or hashcode or toString necessary